supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendSingletonActor.scala

```diff
@@ -1,12 +1,12 @@
 package cromwell.backend.impl.jes
 
 import akka.actor.{Actor, ActorLogging, Props}
-import cromwell.backend.impl.jes.statuspolling.{JesApiQueryManager}
+import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager
 import cromwell.backend.impl.jes.statuspolling.JesApiQueryManager.DoPoll
 
-class JesBackendSingletonActor extends Actor with ActorLogging {
+final case class JesBackendSingletonActor(jesConfiguration: JesConfiguration) extends Actor with ActorLogging {
```

Comment:
Is there a solid reason for that stance though?

I'm happy to be corrected but IMO ADTs should be considered a *data type*, for immutable data storage. This is a mutating, active object. We're instrumenting this actor with a bunch of helper methods for apply, unapply, variable access that we actively don't want and actually go against the grain of how this actor object should be used.